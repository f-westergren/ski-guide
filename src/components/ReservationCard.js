import React from 'react';
import { Card, CardBody, Button } from 'reactstrap';

const ReservationCard = ({ guide_id, first_name, date }) => {
  
  return (
    <Card className="mt-2">
      <CardBody>
        <h6>Reservation with {first_name} on {date}</h6>
        <a className="btn btn-danger float-right ml-2" href="/profile">Cancel</a>      
        <Button className="float-right" color="primary" onClick="">Confirm</Button>       
      </CardBody>
    </Card>
  )
}

export default ReservationCard