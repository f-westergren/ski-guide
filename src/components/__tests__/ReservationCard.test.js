import React from "react";
import { render } from "@testing-library/react";
import ReservationCard from "../ReservationCard";

it("renders without crashing", function() {
  render(<ReservationCard date="2020-10-01" />);
});

it("matches snapshot", function() {
  const { asFragment } = render(<ReservationCard date="2020-10-01" />);
  expect(asFragment()).toMatchSnapshot();
});
