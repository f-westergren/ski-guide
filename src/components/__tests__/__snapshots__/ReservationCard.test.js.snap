// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="mt-2 card"
  >
    <div
      class="card-body"
    >
      <h6>
        Reservation with  on 2020-10-01
      </h6>
      <small
        class="font-italic"
      >
        Awaiting confirmation from Guide
      </small>
      <button
        class="float-right btn btn-danger"
        type="button"
      >
        Cancel
      </button>
    </div>
  </div>
</DocumentFragment>
`;
